fhir-path/src/main/java/com/ibm/fhir/path/function/ValidateCSFunction.java

```diff
@@ -95,17 +95,17 @@ private boolean validate(EvaluationContext evaluationContext, CodeSystem codeSys
                     return true;
                 }
             }
-            generateIssue(evaluationContext, IssueSeverity.ERROR, IssueType.CODE_INVALID, "CodeableConcept does not contain a coding element that matches the specified CodeSystem url and/or version", "%terminologies");
+            generateSupplementalWarning(evaluationContext, IssueSeverity.ERROR, IssueType.CODE_INVALID, "CodeableConcept does not contain a coding element that matches the specified CodeSystem url and/or version", "%terminologies");
```

Comment:
Should this be generateErrorDetail() since the issue it's replacing has a severity of ERROR (same comment on lines 104, 108)?

Reply:
yes, it should. updated.